index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My porfolio</title>
+</head>
+<body>
+    <header>
+        <div class="nav-container">
+            <nav>
```

Comment:
- [ ] Kindly wrap the menu into a nav tag
- And the Logo which is your name in a tag separately don't put both in the nav tag is wrong

Reply:
boss i dont get it can please explain further because i only have 1 review slot